### DISCUSSION TITLE
Type "Date" is neither a built-in type, nor refers to another model, custom type, or enum.

### DISCUSSION THREAD

**Discussion by zwl1619** [prisma/prisma#21911] [Q&A] [answered]
### Question

I use "Date" in Prisma schema， as follow:
```prisma
model User {
  id           String @id @default(auto()) @map("_id") @db.ObjectId
  username     String
  dateOfBirth  Date
}
```

vscode gives an error:
```
Type "Date" is neither a built-in type, nor refers to another model, custom type, or enum.
```
but there is "Date" in Prisma docs: https://www.prisma.io/docs/concepts/database-connectors/mongodb#native-type-mapping-from-prisma-to-mongodb

![Snipaste_2023-11-13_12-09-19](https://github.com/prisma/prisma/assets/13177997/db97c569-4fff-4a23-8741-3d3634e6b2ba)

What should I do?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @zwl1619 👋

Thank you for raising this question 

In Prisma, the `Date` type is not a built-in scalar type. The built-in scalar types in Prisma are String, Boolean, Int, Float, BigInt, Decimal, DateTime, Json, Bytes, and Unsupported.

- https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#datetime

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
